extensions/applicationinsights-angular-js/test-app/src/app/app.component.spec.ts

```diff
@@ -92,23 +81,30 @@ describe('Router: App', () => {
         },
         }
     };
+    
     core.initialize(config, [angularPlugin, analyticsExtension, channel]);
-    // spy on track
-    const spy = spyOn(angularPlugin, 'trackPageView');
```

Comment:
spy should be declared before core.initialize, move spy declaration up to init() function